app/src/main/res/layout/itemdescription_listitem.xml

```diff
@@ -1,69 +1,93 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<RelativeLayout
+<androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:paddingTop="8dp"
     android:paddingLeft="16dp"
     android:paddingRight="16dp"
-    android:paddingBottom="8dp"
-    tools:background="@android:color/holo_orange_light">
+    android:paddingBottom="8dp">
 
     <TextView
         android:id="@+id/txtvPubDate"
         style="@android:style/TextAppearance.Small"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_alignParentRight="true"
-        android:layout_alignParentEnd="true"
         android:layout_alignParentTop="true"
-        android:layout_marginLeft="8dp"
-        android:layout_marginStart="8dp"
-        android:textSize="14sp"
-        android:textColor="?android:textColorSecondary"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true"
+        android:layout_marginStart="4dp"
+        android:layout_marginTop="2dp"
         android:ellipsize="end"
         android:lines="1"
+        android:textColor="?android:textColorSecondary"
+        android:textSize="@dimen/text_size_descriptionsmall"
```

Comment:
Use style, isn't this the same size as the description text? Google has a `body medium` style for that, which you can take as a base